src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5153,6 +5153,12 @@ void func_8083AF44(PlayState* play, Player* this, s32 magicSpell) {
     func_80835DE4(play, this, func_808507F4, 0);
 
     this->unk_84F = magicSpell - 3;
+
+    //! @bug `MAGIC_CONSUME_WAIT_PREVIEW` is not guaranteed to succeed. Ideally, the return value of
+    //! `Magic_RequestChange` should be checked before allowing the process of using a spell to continue.
+    //! If the magic state change request fails, `magicTarget` will never be set correctly.
```

Comment:
```suggestion
    //! If the magic state change request fails, `gSaveContext.magicTarget` will never be set correctly.
```
possibly?

Reply:
I suppose, I feel like its just code stuff that doesnt add anything to the explanation in this case, when im more interested in conveying the meaning here

Reply:
It just feels easier to read through this way, rather than wondering if there's some local variable named `magicTarget` that you should be in the know about